### PR TITLE
Fumo

### PR DESCRIPTION
**Pull request by Jackofashepard**
## About The Pull Request

Fumo
image

## Why It's Good For The Game

Fumo

## Changelog
:cl:
add: Fumo
/:cl:

### CODE REVIEW

**Comment by TheShown911** [comment]
No, no more.

**Comment by Wouju** [comment]
God is dead. God wemains dead. And we have kiwwed him. How shaww we comfowt ouwsewves, the mwuwdewews of aww mwuwdewews?? what was howiest and mwightiest of aww that the wowwd has yet owned has bwed to death undew ouw knives: who wiww wipe this bwood off us?? what watew is thewe fow us to cwean ouwsewves?? what festivaws of atonement, what sacwed games shaww we have to invent?? is not the gweatness of this deed too gweat fow us?? mwust we ouwsewves not become gods simpwy to appeaw wowthy of it?

**Comment by TheShown911** [comment]
Wait, I got a thing for this.

**Comment by Alanii** [comment]
Adding more Fumo is very good for game. I like.

**Comment by TheShown911** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items spawn, thirdly, one and possibly more items are an affront to god and a direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Jackofashepard** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items spawn, thirdly, one and possibly more items are an affront to god and a direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by Medcar** [comment]
image

**Comment by Wouju** [comment]
image
intended file names or did you do a fucky wucky

**Comment by Jackofashepard** [comment]
intended

**Comment by Wouju** [comment]
> intended

i worry it will cause some fuckery for someone later when they look at the code

nice

**Comment by Jackofashepard** [comment]
Wait, lol i mixed them

**Comment by Wouju** [comment]
> Wait, lol i mixed them

Kek

**Comment by CeUvi** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items spawn, thirdly, one and possibly more items are an affront to god and a direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -606,12 +606,24 @@
 	icon_state = "beepskyplushie"
 	phrase = "Ping!"
 
-/obj/structure/plushie/fumo
-	name = "Fumo"
+/obj/structure/plushie/marisafumo
+	name = "Marisa Fumo"
 	desc = "A plushie of a....?."
```
Comment (hyperioo) [inline code/game/objects/items/toys.dm:611]:
```suggestion
	desc = "A plushie of a....?"
```

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -606,12 +606,24 @@
...
-	icon_state = "fumoplushie"
+	icon_state = "marfumoplushie"
 	phrase = "I just don't think about losing."
 
+/obj/structure/plushie/cirnofumo
+	name = "Cirno Fumo"
+	desc = "A plushie of a.... where are these coming from?."
```
Comment (hyperioo) [inline code/game/objects/items/toys.dm:617]:
```suggestion
	desc = "A plushie of a.... where are these coming from?"
```

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -606,12 +606,24 @@
...
+	icon_state = "cirfumoplushie"
+	phrase = "I can even freeze magicians!"
+
+/obj/structure/plushie/astolfofumo
+	name = "Astolfo Fumo"
+	desc = "A plushie of a.... wait is that a bulge?."
```
Comment (hyperioo) [inline code/game/objects/items/toys.dm:623]:
```suggestion
	desc = "A plushie of a.... wait is that a bulge?"
```

**Review by hyperioo — CHANGES_REQUESTED** [review]
No reason to have these `.` after `?`

**Comment by Honestly101** [comment]
there is no god here, this is all pure cringe.

**Comment by Omni44** [comment]
300px-Creating_Bugs_Bunny's__No_

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -606,12 +606,24 @@
...
 	phrase = "I just don't think about losing."
 
+/obj/structure/plushie/cirnofumo
+	name = "Cirno Fumo"
+	desc = "A plushie of a.... where are these coming from?"
+	icon_state = "cirfumoplushie"
+	phrase = "I can even freeze magicians!"
+
+/obj/structure/plushie/astolfofumo
+	name = "Astolfo Fumo"
```
[the remark this answers is not in the document]
Comment (Wouju) [reply code/game/objects/items/toys.dm:622]: When you code it

**Comment by Firefox13** [comment]
We all know that every second person playing Eris is a weeb and just one fumo is enough to prove it. Please execute some restrain when adding memes. Especially fucking Astolfo.

**Comment by AlexMorgan3817** [comment]
> We all know that every second person playing Eris is a weeb and just one fumo is enough to prove it. Please execute some restrain when adding memes. Especially fucking Astolfo.

https://youtu.be/8zEiwC0jy78

**Pull request closed without merging**